common/src/main/java/tachyon/util/CommonUtils.java

```diff
@@ -157,27 +157,41 @@ public static ByteBuffer cloneByteBuffer(ByteBuffer buf) {
    * Join each element in paths in order, separated by {@code TachyonURI.SEPARATOR}.
    * <p>
    * For example, {@code concatPath("/myroot/", "dir", 1L, "filename")} returns
-   * {@code "/myroot/dir/1/filename"}
+   * {@code "/myroot/dir/1/filename"}, or
+   * {@code concatPath("tachyon://myroot", "dir", "filename")} returns
+   * {@code "tachyon://myroot/dir/filename"}.
    *
    * @param paths to concatenate
    * @return joined path
    */
   public static String concatPath(Object... paths) {
     List<String> trimmedPathList = new ArrayList<String>();
-    for (int k = 0; k < paths.length; k ++) {
+    String joinedPath;
+    String prefix = null;
```

Comment:
thanks, the assignment "prefix = null" can be moved from initialization to a single line, but it must be set as a flag.